cli/pkg/helm/helm.go

```diff
@@ -1,4 +1,4 @@
-package create
+package helm
```

Comment:
Moved this code in a separate package outside of `cli/pkg/cmd/create`, so that we can re-use it.